Formula/lighthouse.rb

```diff
@@ -0,0 +1,29 @@
+class Lighthouse < Formula
+  desc "Rust Ethereum 2.0 Client"
+  homepage "https://github.com/sigp/lighthouse"
+  url "https://github.com/sigp/lighthouse/archive/refs/tags/v2.0.1.tar.gz"
+  sha256 "b1ae766c0b335e963d7ebd6ab2a02386078a6b2ab688ec5da3604191a4d0d3f4"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "zlib"
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./#{lighthouse}")
```

Comment:
I assume you mean this, since there is no `lighthouse` variable
```suggestion
    system "cargo", "install", *std_cargo_args(path: "./lighthouse")
```